medallion/__init__.py

```diff
@@ -67,3 +81,14 @@ def register_blueprints(flask_application_instance):
         current_app.register_blueprint(discovery.mod)
         current_app.register_blueprint(manifest.mod)
         current_app.register_blueprint(objects.mod)
+
+
+@application_instance.errorhandler(500)
+def handle_error(error):
+    error = {
+        "title": error.args[0],
+        "http_status": "500"
+    }
+    return Response(response=flask.json.dumps(error),
+                    status=500,
+                    mimetype=MEDIA_TYPE_STIX_V20)
```

Comment:
Since this is returning a TAXII error, it should use `medallion.views.MEDIA_TYPE_TAXII_V20`.

I actually did something like this [here](https://github.com/gtback/cti-taxii-server/blob/6498a3bbe7d7422551778ca5ec4bba38e4e89994/medallion/__init__.py#L74-L80), but haven't merged it in yet.

Reply:
Good catch! Thanks for pointing that out.